### ISSUE TITLE
sparkle updates broken

### ISSUE THREAD

**Issue by core-code** [hsoft/moneyguru#500]
automatic updates from 
2.10.2 
to
2.11.0

don't work on macOS

the appcast doesn't even contain the 2.11 release

**Comment by core-code** [comment]
additionally the current release has weird suffixed '-1' in the CFBundleVersion and CFBundleShortVersionString entries, which is incompatible with Sparkle

**Comment by hsoft** [comment]
https://github.com/hsoft/moneyguru#os-x-maintainer-wanted has been there for a while, but I'll have to update it: I don't have a mac any more. Unless someone steps up (@markreds81 seemed interested), broken mac stuff will stay broken.

**Comment by markreds81** [comment]
Yes, I confirm my interest on it. I've started to take a look at the moneyguru-cocoa repository which is the official source code repository for the Mac version of the application.
